### ISSUE TITLE
[Accessibility] User unable to navigate drop down items using up or down arrow key in scan mode.

### ISSUE THREAD

**Issue by rishabjainscorpio** [microsoft/fluentui#18832]
### Environment Information

- **Package version(s)**: 9.6.1-fluent2
- **Browser and OS versions**: Edge and Chrome

**Comment by smhigley** [comment]
@rishabjainscorpio selecting an item from a combobox's dropdown isn't expected to work in scan mode/virtual cursor. Even the native `<select>` element's dropdown isn't really navigable with virtual cursor. In order to actually change the value of a form field, users are expected to use forms mode (hence the name).

Could you elaborate on where the expectation is coming from, and provide a link and more detailed description?
